gdpr/gdpr.go

```diff
@@ -29,9 +29,10 @@ type Permissions interface {
 	AMPException() bool
 }
 
+// Versions of the GDPR TCF technical specification.
 const (
-	tCF1 uint8 = 1
-	tCF2 uint8 = 2
+	tcf1SpecVersion uint8 = 1
```

Comment:
I really don't like `tcf1SpecVersion`. Other possibilities are `tcf1Version`, `tcf1Spec`, `tcfSpecVersion`, `tcfSpec1`. I can't use just `tcf` due to package name collisions.

Reply:
There doesn't seem to be a good solution here unfortunately.